pkg/deploy/controller/generictrigger/controller_test.go

```diff
@@ -149,6 +149,19 @@ func TestHandle_changeWithTemplateDiff(t *testing.T) {
 		config := testapi.OkDeploymentConfig(1)
 		config.Namespace = kapi.NamespaceDefault
 		config.Spec.Triggers = []deployapi.DeploymentTriggerPolicy{testapi.OkConfigChangeTrigger()}
+
+		versioned, err := kapi.Scheme.ConvertToVersion(config, deployv1.SchemeGroupVersion)
```

Comment:
Can you pack roundtripping inside a helper under pkg/deploy/api/test? I came across this issue in https://github.com/openshift/origin/pull/9349 and I will need to reuse it.